build.js

```diff
@@ -1,12 +1,19 @@
 import {build} from 'esbuild';
 
-build({
-    entryPoints: ['src/index.ts'],
-    external: ['antlr4ng', 'antlr4-c3'],
-    bundle: true,
-    minify: true,
-    keepNames: true,
-    format: 'esm',
-    outfile: 'dist/index.js',
-    tsconfig: './tsconfig.build.json',
-});
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
Are you sure there's no way to do write it differently? If not, at least write a comment explaining why you need to disable it.

Reply:
the script has been rewritten, there is no more comment